### PR TITLE
Remove Brigade

### PR DESCRIPTION
**Pull request by meyerbaptiste**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A
| License       | MIT
| Doc PR        | N/A

This is a money fix 💸

### CODE REVIEW

**Comment by sroze** [comment]
Hipster 😄

**Comment by meyerbaptiste** [comment]
This PR is sponsored by @lenancker!

**Comment by meyerbaptiste** [comment]
@dunglas Is it possible to disable the webhook for the master branch?

File: .travis.yml [hunk 1]
```diff
@@ -41,19 +41,76 @@ before_install:
...
+  - >
+    if [[ $coverage = 1 ]]; then
+      phpdbg -qrr vendor/bin/phpunit --coverage-php build/cov/coverage-phpunit.cov;
+    else
+      vendor/bin/phpunit;
+    fi
+  - >
+    if [[ $coverage = 1 ]]; then
+      for f in $(find features -name '*.feature'); do
+        if [[ $f != 'features/main/exposed_state.feature' ]]; then
```
[the remark this answers is not in the document]
Comment (Simperfit) [reply .travis.yml]: `-not -path features/main/exposed_state.feature` in the find ?

[the remark this answers is not in the document]
Comment (meyerbaptiste) [reply .travis.yml]: I meant instead of hardcoding the filename.

**Comment by dunglas** [comment]
Can you target 2.2 now @meyerbaptiste please?

**Comment by dunglas** [comment]
Bridgade hook disabled.

**Comment by meyerbaptiste** [comment]
Done @dunglas.

**Comment by dunglas** [comment]
Thanks @meyerbaptiste!

**Pull request merged**